packages/wallets/src/features/cashier/modules/Transfer/hooks/useTransferMessages/utils/cumulativeAccountLimitsMessageFn.ts

```diff
@@ -40,8 +40,8 @@ const cumulativeAccountLimitsMessageFn = ({
         !targetAccount.currency ||
         !sourceAccount.currencyConfig ||
         !targetAccount.currencyConfig ||
-        !allowedSumUSD ||
-        !availableSumUSD
+        (allowedSumUSD !== 0 && !allowedSumUSD) ||
+        (availableSumUSD !== 0 && !availableSumUSD)
```

Comment:
Updated this condition to return null when value is falsy but not 0 as correct message will not be returned when available transfers for `daily_cumulative_amount_transfers` has reached 0
<img width="920" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/3f185d33-2120-4c59-8a48-c2a447674b41">